### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by Rickyxstar**
# Why are we making this change?

This change aims to enhance the stability of the Thoras platform by introducing unit testing for the Thoras Helm Chart.

The addition of unit tests will provide comprehensive validation of the Thoras Helm Chart, ensuring its integrity and preventing potential issues that could arise during deployment.

# Implementation Details

Unit tests have been implemented and will be executed on every pull request to the main branch. The tests are run using the `helm-unittest` plugin, ensuring a consistent and reliable testing process.

**Test Execution:**

The following command is used to run the unit tests:

```bash
helm plugin install https://github.com/helm-unittest/helm-unittest.git
helm unittest ./charts/thoras --chart-tests-path ./charts/thoras/tests
```

### Expected Benefits

* **Increased Stability:** Early detection and prevention of potential issues through comprehensive testing.
* **Improved Reliability:** Confidence in the Thoras Helm Chart's functionality and consistent behavior.
* **Reduced Deployment Risks:** Mitigation of deployment-related problems due to improved chart integrity.

This enhancement demonstrates our commitment to delivering a robust and stable Thoras platform.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    branches:
+      - main
+
+jobs:
+  helm-unit-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Install Helm
```
Comment (dspigarelli) [inline .github/workflows/ci.yml:17]: [dust] I'm coming at this with completely new eyes, but the `curl` made me wonder if there's an existing action that could install helm. Would the following work?

```yaml
     -  uses: azure/setup-helm@v4.2.0
        with:
          version: v3.14.4
```

Comment (dspigarelli) [reply .github/workflows/ci.yml:17]: BTW, I got this from https://github.com/helm/chart-testing-action

Comment (Rickyxstar) [reply .github/workflows/ci.yml]: I've updated the step to use the install helm action from azure!

**Review by herrewig — COMMENTED** [review]
Let's goooo!!! 🔥

**Pull request merged**